gtk/pyproject.toml

```diff
@@ -64,7 +64,7 @@ root = ".."
 [tool.setuptools_dynamic_dependencies]
 dependencies = [
     "pycairo >= 1.17.0",
-    "pygobject >= 3.50.0",
+    "pygobject == 3.50.0",
```

Comment:
There should be a comment explaining this pin and linking to the relevant issue.